docs/content/config/advanced/mail-forwarding/gmail-smtp.md

```diff
@@ -0,0 +1,52 @@
+---
+title: 'Mail Forwarding | GMAIL SMTP'
+---
+
+[GMAIL SMTP (Simple Message Transport Protocol)][gmail-smtp] provides a simple way for cloud based applications to send and receive email.
+
+!!! example "Configuration via ENV"
+
+    [Configure a relay host in DMS][docs::relay] to forward all your mail through GMAIL SMTP:
+
+    - `RELAY_HOST` should match your [SMTP Server Endpoint][gmail-smtp::relay-host].
+    - `RELAY_PORT` should be set to [one of the supported Gmail SMTP ports][gmail-smtp::relay-port] (_eg: 587 for STARTTLS_).
+    - `RELAY_USER` and `RELAY_PASSWORD` should be set to your [Gmail Account ID][gmail-smtp::account-id].
+
+    ```env
+    RELAY_HOST=smtp.gmail.com
+    RELAY_PORT=587
+    # Alternative to RELAY_HOST + RELAY_PORT which is compatible with LDAP:
+    DEFAULT_RELAY_HOST=[smtp.gmail.com]:587
+
+    RELAY_USER=someone@gmail.com
+    RELAY_PASSWORD=xxx
+    ```
+
+!!! warning "Process of providing RELAY_PASSWORD"
+
+    You should use your [2-step verification app password][gmail-smtp::2-step-password], **not** your gmail account password.
+    `setup relay add-auth` is a better alternative, which manages the credentials via a config file.
```

Comment:
Ohhhh ok. So what is the value from the earlier Gmail Account ID link? (_not accessible if you're not signed in_)

You are definitely going for the personal Gmail here and this admonition is good to raise awareness that it should be an App Password, not the users personal credentials. I think that extra clarity should be added to the earlier example with link instead of "Gmail Account ID"?